### PR TITLE
Add a bio file for GitHub Teacher

### PR DESCRIPTION
**Pull request by githubteacher**
This Pull Request adds a bio file for @githubteacher, as requested in #6.

@githubstudent, I would :heart: if you had some :watch: to put some :eyes: on this and let me know your :thought_balloon:.

### CODE REVIEW

**Comment by githubteacher** [comment]
This looks good for the most part.  But there are a few things that need to be addressed.

File: docs/githubteacher.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# GitHub Teacher Bio
+
+- **Name:** GitHub Teacher
+- **Occupation:** Teacher
+- **Location:** San Francisco
```
Comment (githubteacher) [inline docs/githubteacher.md]: What if you were more specific here?  There are lots of San Franciscos in the :earth_africa:

Comment (pandasla) [inline docs/githubteacher.md]: Are you in California?

**Comment by githubteacher** [comment]
Looks good to me!
:+1: 
:ship: it!
:shipit:

**Pull request merged**